apps/unsubscribe-automation/src/main.ts

```
@@ -0,0 +1,242 @@
+  confirmationIndicator: z.string().nullable(),
+});
+
+type PageAnalysis = z.infer<typeof pageAnalysisSchema>;
+
+async function analyzePageWithAI(pageContent: string): Promise<PageAnalysis> {
+  const maxContentLength = 20000; // Adjust based on API limitations
+  if (pageContent.length > maxContentLength) {
+    console.warn(
+      `Page content exceeds ${maxContentLength} characters. Truncating...`,
+    );
+    pageContent = pageContent.substring(0, maxContentLength);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)